internal/pkg/service/stream/definition/repository/branch_limits_test.go

```diff
@@ -1,4 +1,4 @@
-package repository
+package repository_test
 
 import (
 	"context"
```

Comment:
I prefer blackbox testing using `repository_test` pkg, instead of whitebox testing using `repository` pkg.
  - It also prevents some types of import cycle issues.
  - I updated the remaining tests in the Stream Service repositories.